pyswmm/errors.py

```diff
@@ -0,0 +1,22 @@
+class IncompleteSimulation(Exception):
+    """Exception raised for incomplete simulation
+
+    Attributes:
+        message -- explanation of the error
+    """
+
+    def __init__(self, message):
+        self.message = message
+        super().__init__(self.message)
+
+
+class OutputException(Exception):
```

Comment:
Why carry a custom exception class when swmm-toolkit already provides one?

Reply:
The intention of creating OutputException was to provide a more detailed explanation when interacting with the output file.

https://github.com/jennwuu/pyswmm/blob/648b463582930512a63bbaddc2a3458517ce05d7/pyswmm/output.py#L54-L108

Reply:
Sure but swmm-toolkit already performed these checks. swmm-toolkit does verification on the arguments passed and throws exceptions containing the error message. So this is somewhat duplicative of that functionality. 

Don't Repeat Yourself - DRY

Always strive to get maximum functionality from the fewest lines of code possible.

Reply:
This is something I would let the wrapper handle.

Reply:
Ask yourself, What is the purpose of this API? How can you add value to the lower level API that swmm-toolkit provides? Something that would make this a lot easier to use is hiding the notion of an "index". Node link indexes are not really that useful, so why not abstract them away? Reporting period index is also confusing. Wouldn't it be better to use element IDs and dates and times instead of indexes?

Reply:
That makes sense to me. I will update this PR to remove the duplicated checks and surface the error from swmm-toolkit.